[BUG] Duplicate trello field causes mypy typecheck failure

### Summary

Running `make typecheck` fails due to a duplicate `trello` field definition in `EffectiveIntegrations`.

<img width="845" height="130" alt="Image" src="https://github.com/user-attachments/assets/278d5a98-63a8-40a3-8dd8-5285042dd337" />

### Expected vs actual behavior

**Expected**:

`make typecheck` should pass successfully on a fresh setup.

**Actual Behavior**:
mypy throws the following error:
app/integrations/models.py:446: error: Name "trello" already defined on line 443 [no-redef]


### Steps to reproduce

1. Clone the repository
2. Set up virtual environment and install dependencies
3. Run:
python -m venv .venv
source .venv/bin/activate
pip install -e ".[dev]"
make lint && make typecheck && make test-cov

### Can you reproduce it consistently?

Yes

### How often does it occur?

Every time

### Operating system

Linux

### Logs and error output

_No response_

### Additional context

_No response_

--- a/app/integrations/models.py
+++ b/app/integrations/models.py
@@ -443,4 +443,3 @@ class EffectiveIntegrations(StrictConfigModel):
     trello: EffectiveIntegrationEntry | None = None
     discord: EffectiveIntegrationEntry | None = None
     mysql: EffectiveIntegrationEntry | None = None
-    trello: EffectiveIntegrationEntry | None = None

